app/views.py

```diff
@@ -36,5 +40,17 @@ def terms_and_conditions(request):
     return render(request, 'terms_and_conditions.html')
 
 
+def protectedMedia(request, file):
+    document = get_object_or_404(Application, resume=file)
+    path, file_name = os.path.split(file)
+    if request.user.is_authenticated() and (request.user.is_organizer or
+                                            (document and (document.user_id == request.user.id))):
+        response = FileResponse(document.resume)
+        response["Content-Type"] = ""
```

Comment:
Return type so that it can be previewed on the browser. Specially useful for `pdf` and images.

See: https://stackoverflow.com/questions/312230/proper-mime-media-type-for-pdf-files

Reply:
There are more files rather that PDFs, imatges or Word documents, I left that empty do that the browser would guess the type. Previous version didn't allow PDF visualization on the browser.

Reply:
It did allow it

https://my.hackcu.org/files/resumes/resume_wJeCzht.pdf

Reply:
Doesn't work for me, PDF downloads immediately (tested on Firefox and Chrome both Windows and Linux).